tech/languages/java/java-build-tools-installation.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Java Build Tools
+page: java
+---
+
+## Maven installation
+
+Maven is probably the most popular build tool in Java world these days. It's available in official Fedora repositories, in form of RPM. To install it, simply type:
+
+```
+# dnf install maven
+```
```

Comment:
I am not really opposed to the idea of briefly mentioning how to use the build tools, but it's not really Fedora-specific. Maybe mentioning that after successful installation, the "mvn" command will be available on the system? I guess that the Fedora developer portal shouldn't be about teaching users how to use Maven, right?